scripts/climate

```diff
@@ -120,12 +121,11 @@ if ($cover =~ /coverage:\s?(.+)%/) {
         }
         $error = 1;
     } else {
-        print "The tests are covering the $fl% of this package!\n";
+        print "The tests are covering $fl% of this package!\n";
     }
     if ($opts{'o'}) {
         `go tool cover -html=c.out`;
     }
-    `rm -f c.out`;
```

Comment:
It is still produced. The intention is not to delete it so that we can check the profile afterwards. I figured that it would be better to add another option to avoid deletion?